freqtrade/strategy/resolver.py

```diff
@@ -78,6 +82,14 @@ def _load_strategy(
             # Add extra strategy directory on top of search paths
             abs_paths.insert(0, extra_dir)
 
+        if validators.url(strategy_name):
+            temp = tempfile.mkdtemp("freq", "strategy")
+            abs_paths.insert(0, temp)
+            name = os.path.basename(urlparse(strategy_name).path)
+            urlretrieve(strategy_name, os.path.join(temp, name))
```

Comment:
why not use requests ... ? i think that's cleaner to download files ...

Reply:
requests is based on urllib, no reason to use something refined, if urlretrieve, which comes with urllib does the job already in an easy and clean way.

Reply:
+1 for using requests. Also the content could be passed directly instead of creating a temp file. The code could look something like that
```
resp = requests.get(url, stream=True)
if r.status_code == 200:
    r.raw.decode_content = True
    strategy = resp.decode('utf-8')
```

Reply:
switched to requests in my local branch, I do like the idea to keep it as a temp file for ease of debugging and doesn't cost much.